go/stats/timings.go

```diff
@@ -18,13 +18,16 @@ package stats
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
 	"sync"
 	"time"
 
 	"vitess.io/vitess/go/sync2"
 )
 
+var timingsBufferSize = flag.Int("timings_buffer_size", 5, "RingFloat64 capacity used by Timings stats")
```

Comment:
I thought about adding a parameter like `bufferSize` to `NewTimings()` but we would need to pass in a flag the all the callsite of `stats.NewTimings`

I couldn't come up with a good way to make the buffer size configurable through flag while also making it non-zero during the test, so I end up making the default size to be 5 here so that we have something available for test while does not consume too many unnecessary memories in production. But please let me know if there is any concern or better solution here